tests/test_linearGaussian_apt.py

```diff
@@ -17,18 +17,21 @@
 # running all combinations is excessive. The standard test is (3, "gaussian", "snpe_c"),
 # and we then vary only one parameter at a time to test single-d, uniform, and snpe-b
 @pytest.mark.parametrize(
-    "num_dim, prior_str, algorithm_str",
+    "num_dim, prior_str, algorithm_str, simulation_batch_size",
     (
-        (3, "gaussian", "snpe_c"),
-        (1, "gaussian", "snpe_c"),
-        (3, "uniform",  "snpe_c"),
-        (3, "gaussian", "snpe_b"),
+        (3, "gaussian", "snpe_c", 10),
+        (1, "gaussian", "snpe_c", 10),
+        (3, "uniform", "snpe_c", 10),
+        (3, "gaussian", "snpe_b", 10),
+        (3, "gaussian", "snpe_c", 1),
     ),
 )
-def test_apt_on_linearGaussian_based_on_mmd(num_dim: int, prior_str: str, algorithm_str: str):
+def test_apt_on_linearGaussian_based_on_mmd(
```

Comment:
If this is still in the danger of failing stochastically, we can set a fixed seed